service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/service/EncounterReportBundleService.java

```diff
@@ -55,6 +59,7 @@ public Bundle createEncounterBundle(POCDMT000002UK01ClinicalDocument1 clinicalDo
         ListResource listResource = listMapper.mapList(clinicalDocument, encounter, resourcesCreated);
 
         addList(bundle, listResource);
+        addCarePlan(bundle, carePlans);
```

Comment:
This must be add before list, as this is a list of all resources

Reply:
I did and the integration test failed

Reply:
list must come last or careplan wont exist in list of all resources

Reply:
Fixed...